spec/v1/envelope.md

```diff
@@ -19,5 +21,41 @@ standard DSSE fields.
     specifying its schema.
 -   `payload` MUST be a base64-encoded JSON [Statement].
 
+## File naming convention
+
+Envelopes SHOULD use the suffix `.json`.
```

Comment:
I wonder if this section should say something like "if stored in a file by itself, and not in a [Bundle], ..."

Reply:
done.